controllers/preflightvalidation_controller.go

```diff
@@ -0,0 +1,239 @@
+/*
```

Comment:
All other controllers in `controllers/` are named `<CR name>_reconciler.go`. Should we rename?

Reply:
fixed